examples/tests/sdk_tests/expect/assert.test.w

```diff
@@ -169,6 +175,14 @@ let myClass = new MyClass(1, "hello");
 expect.equal(myClass, myClass);
 expect.notEqual(myClass, new MyClass(1, "hello world") as "yet another my class");
 
+test "fail test" {
+   try {
+    expect.fail("This is a failure");
+   }catch err{
+    expect.equal(err, "This is a failure");
+   }
+}
```

Comment:
Can we add some failing cases for the other new methods (checking that an error is thrown when `expect.match` fails or `expect.ok` fails)?

Reply:
@Chriscbr Addressed the comment